app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryRanking.tsx

```diff
@@ -15,7 +16,7 @@ interface TooltipProps {
 const Tooltip = ({ text }: TooltipProps) => {
   return (
     <div className="bg-blue-100 border-b border-gray-450 flex mb-4 py-4">
-      <Icon className="mx-4" size="medium" symbol="info" />
+      <Icon className="mx-xf4" size="medium" symbol="info" />
```

Comment:
What does this do?

Reply:
ha good catch! I think it's a key command that accidentally got added.